### ISSUE TITLE
Allocate IPv6 resources 

### ISSUE THREAD

**Issue by swift2plunder** [Pterodactyl/Panel#62]
When attempting to allocate IPv6 resources on a node, the panel currently claims victory and silently fails.

**Comment by DaneEveritt** [comment]
This will need to be further investigated to implement properly with docker.

**Comment by DaneEveritt** [comment]
Hey @swift2plunder, just getting back to this issue for the next release. I realized I didn't get a lot of information from this and probably misread it.

From what I understand you're saying you can try to add a v6 address, and the panel will say it was added, but really it isn't and nothing new happens? Or is i that you can add the IP and suh, but when allocating it to a server it fails?

**Comment by swift2plunder** [comment]
I never got to the point of investigating what may or may not have happened in the daemon. The extent of my observation was that the panel gave a success message, but didn't show the IPV6 resources in subsequent lists of allocations. What went over the wire and what the daemon may or may not have attempted to do are separate but related issues.

**Comment by swift2plunder** [comment]
It should be noted that last I checked, Minecraft acts funny on IPV6 enabled systems. Java likes to glom onto the IPV6 addresses available for listening, but the client won't accept IPV6 addresses as valid for connecting. The command line switch to have Java connect to IPV4 addresses doesn't seem to work for me and I've yet to successfully configure a bridge using either Linux network tools or socat.

I guess I'm saying that when I filled this bug I was naive about Minecraft's ability to serve on an IPV6 network.

Also my use case of running servers on cheap IPV6 connected hardware and routing the traffic through some hardware where IPV4 addresses are more plentiful has proven impractical.
